### PR TITLE
Bring Back Session Upload Event to 4.4

### PR DESCRIPTION
**Pull request by quinqu**
Backport of a fix from master for bringing back session upload event

Original fix against master: [#5470](https://github.com/gravitational/teleport/pull/5470)

### CODE REVIEW

**Comment by awly** [comment]
@quinqu is this a backport of a fix from `master` or a standalone change?

**Comment by quinqu** [comment]
@awly This is a backport fix

File: lib/events/events.proto [hunk 1]
```diff
@@ -951,6 +951,7 @@ message OneOf {
         events.OIDCConnectorDelete OIDCConnectorDelete = 33;
         events.SAMLConnectorCreate SAMLConnectorCreate = 34;
         events.SAMLConnectorDelete SAMLConnectorDelete = 35;
+        events.SessionUpload SessionUpload = 36;
```
Comment (awly) [inline lib/events/events.proto:954]:
```suggestion
        events.SessionUpload SessionUpload = 43;
```

**Review by russjones — APPROVED** [review]
Bot.

**Comment by russjones** [comment]
@quinqu Please wait for @awly re-review and also don't merge before March 1st.

**Review by awly — APPROVED** [review]

**Comment by russjones** [comment]
Temporarily closing this while issues are addressed.

**Comment by robertogiordani** [comment]
@quinqu in which 4.4.x version is included this fix?

**Comment by quinqu** [comment]
@robertogiordani this will be in Teleport 4.4.9

**Comment by robertogiordani** [comment]
ETA?

Regards,
    Roberto.
________________________________________________________________________________

Roberto Giordani
Infrastructure Technical Security Lead 
IBM Rome Software Lab 
IBM Public Cloud
phone +39.348.4327340
E-mail roberto.giordani@it.ibm.com
WebMeeting https://ibm.webex.com/join/roberto.giordani
IBM Italia S.p.A. Via Luigi Stipa, 150 - 00148 Rome

**Review by awly — APPROVED** [review]

File: lib/srv/exec_test.go [hunk 2]
```diff
@@ -515,3 +515,6 @@ func (a *fakeLog) WaitForDelivery(context.Context) error {
 func (a *fakeLog) Close() error {
 	return trace.NotFound("")
 }
+func (a *fakeLog) EmitAuditEvent(ctx context.Context, e events.AuditEvent) error {
```
Comment (a-palchikov) [inline lib/srv/exec_test.go:518]: Hmm, I don't think fakeServer is even using auditLog - tests pass even if it does not have it so I'd just remove it.

**Review by a-palchikov — APPROVED** [review]

File: lib/srv/exec_test.go (hunk 2 above)
Comment (quinqu) [reply lib/srv/exec_test.go:518]: The tests do not pass if it has it does not have it

**Review by russjones — APPROVED** [review]
Bot.

**Comment by robertogiordani** [comment]
Hi Jane @quinqu and @russjones
we are still waiting the version 4.4.9.
Do you have the exact day when will be released?
Thanks


Regards,
    Roberto.
________________________________________________________________________________

Roberto Giordani
Infrastructure Technical Security Lead 
IBM Rome Software Lab 
IBM Public Cloud
phone +39.348.4327340
E-mail ***@***.***
WebMeeting https://ibm.webex.com/join/roberto.giordani
IBM Italia S.p.A. Via Luigi Stipa, 150 - 00148 Rome










From:   jane quin ***@***.***>
To:     gravitational/teleport ***@***.***>
Cc:     robertogiordani ***@***.***>, Mention 
***@***.***>
Date:   02/03/2021 19:37
Subject:        [EXTERNAL] Re: [gravitational/teleport] Bring Back Session 
Upload Event to 4.4 (#5636)



@robertogiordani this will be in Teleport 4.4.9 — You are receiving this 
because you were mentioned. Reply to this email directly, view it on 
GitHub, or unsubscribe. 
‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍
@robertogiordani this will be in Teleport 4.4.9

**Comment by webvictim** [comment]
@robertogiordani It's out now.

**Comment by robertogiordani** [comment]
Thanks!

Regards,
    Roberto.
________________________________________________________________________________

Roberto Giordani
Infrastructure Technical Security Lead 
IBM Rome Software Lab 
IBM Public Cloud
phone +39.348.4327340
E-mail ***@***.***
WebMeeting https://ibm.webex.com/join/roberto.giordani
IBM Italia S.p.A. Via Luigi Stipa, 150 - 00148 Rome










From:   Gus Luxton ***@***.***>
To:     gravitational/teleport ***@***.***>
Cc:     robertogiordani ***@***.***>, Mention 
***@***.***>
Date:   17/03/2021 22:47
Subject:        [EXTERNAL] Re: [gravitational/teleport] Bring Back Session 
Upload Event to 4.4 (#5636)



@robertogiordani It's out now. — You are receiving this because you were 
mentioned. Reply to this email directly, view it on GitHub, or 
unsubscribe. 
‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍‍
@robertogiordani It's out now.

**Pull request merged**